tamr_client/operation.py

```diff
@@ -68,6 +68,19 @@ def succeeded(operation: Operation) -> bool:
     return operation.status is not None and operation.status["state"] == "SUCCEEDED"
 
 
+def from_resource_id(
+    session: Session, instance: Instance, resource_id: str
+) -> Operation:
+    """Get operation by ID
+
+    Args:
+        resource_id: The ID of the operation
+    """
+    url = URL(instance=instance, path=f"operations/{resource_id}")
+    r = session.get(str(url), headers={"Accept": "application/json"})
```

Comment:
I don't think the `headers` part is necessary given that we don't use that in other `session.get` requests in TC.

Reply:
That's right. I'll drop those.